<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mcaPlots</class>
 <widget class="QDialog" name="mcaPlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic MCA Viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="scanNumber">
         <property name="text">
          <string>MCA point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mcaID">
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="showPreviousBtn">
         <property name="text">
          <string>Prev</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showNextBtn">
         <property name="text">
          <string>Next</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="PlotWidget" name="viewPart"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetZoom">
       <property name="text">
        <string>Reset zoom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
